aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -394,6 +395,48 @@ public class AwsProperties implements Serializable {
    */
   public static final String S3_WRITE_TAGS_PREFIX = "s3.write.tags.";
 
+  /**
+   * Used by {@link GlueCatalog} to tag objects when writing. To set, we can pass a catalog
+   * property.
+   *
+   * <p>For more details, see
+   * https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-tagging.html
+   *
+   * <p>Example: s3.write.table-name-tag-enabled=true
+   */
+  public static final String S3_WRITE_TABLE_NAME_TAG_ENABLED = "s3.write.table-name-tag-enabled";
```

Comment:
I am wondering if we should simplify these config names to `s3.write.table-tag-enabled`, `s3.write.namespace-tag-enabled`, `iceberg.table`, `iceberg.namespace`, what do you think?

Reply:
Yes, seems clean to me. I have made the changes.